### PR TITLE
[feature8055] New feature is added.

### PR DESCRIPTION
**Pull request by RishabhShukla1626**
added a new file with a new feature.

### CODE REVIEW

**Review by mikegithubber — CHANGES_REQUESTED** [review]
Add extension .txt to the file and put it to the folder feature8055

File: feature-added [hunk 1]
```diff
@@ -0,0 +1 @@
+New feature is added to this project.
```
Comment (Anup9661) [inline feature-added:1]: this code is nicely work.

**Review by Anup9661 — APPROVED** [review]
good job mike.

File: feature-added (hunk 1 above)
Comment (rolandlitzba) [inline feature-added:1]: It's really good that you have decided to add this line of code

**Comment by dabbott12** [comment]
> added a new file with a new feature.

Can you be more specific? Which file did you add and for which feature?

Thanks

**Review by dabbott12 — CHANGES_REQUESTED** [review]
Please be specific on which feature was added.

**Review by dean3772 — APPROVED** [review]
Good job mike!

**Review by EugeneCNC — CHANGES_REQUESTED** [review]
Will be great to add a functionality description.

**Pull request closed without merging**